_spinnaker/operator-config.md

```diff
@@ -3,90 +3,261 @@ layout: post
+    files: {}
+  #      profiles__rosco__packer__example-packer-config.json: |
+  #        {
+  #          "packerSetting": "someValue"
+  #        }
+  #      profiles__rosco__packer__my_custom_script.sh: |
+  #        #!/bin/bash -e
+  #        echo "hello world!"
+
+
+  # spec.expose - This section defines how Spinnaker should be publicly exposed.
   expose:
-    type: service
+    type: service  # Kubernetes LoadBalancer type (service/ingress). Note that only "service" is supported for now.
     service:
-      type: [kubernetes service type]
+      type: LoadBalancer
+
+      # Annotations to be set on Kubernetes LoadBalancer type.
+      # They only apply to spin-gate, spin-gate-x509, or spin-deck.
       annotations:
-        [map of key:value pairs]
-      publicPort: [integer]
-      overrides:
-        [service name]:
-          type: [kubernetes service type]
-          annotations:
-            [map of key:value pairs]
+        service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
+        # Uncomment the line below to provide an AWS SSL certificate to terminate SSL at the LoadBalancer.
+        #service.beta.kubernetes.io/aws-load-balancer-ssl-cert: arn:aws:acm:us-west-2:9999999:certificate/abc-123-abc
+
+      # Provide an override to the exposing KubernetesService.
+      overrides: {}
+      # The following example is an example config for the Gate-X509 configuration.
+#        deck:
+#          annotations:
+#            service.beta.kubernetes.io/aws-load-balancer-ssl-cert: arn:aws:acm:us-west-2:9999999:certificate/abc-123-abc
+#            service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
+#        gate:
+#          annotations:
+#            service.beta.kubernetes.io/aws-load-balancer-ssl-cert: arn:aws:acm:us-west-2:9999999:certificate/abc-123-abc
+#            service.beta.kubernetes.io/aws-load-balancer-backend-protocol: https  # X509 requires https from LoadBalancer -> Gate
+#       gate-x509:
+#         annotations:
+#           service.beta.kubernetes.io/aws-load-balancer-backend-protocol: tcp
+#           service.beta.kubernetes.io/aws-load-balancer-ssl-cert: null
+#         publicPort: 443
+
+  validation:
+```
+### metadata.name
+
+Name of your Spinnaker service. Use this name to view, edit, or delete Spinnaker. The following example uses the name `prod`:
+
+```bash
+$ kubectl get spinsvc prod
 ```
 
-**spec.spinnakerConfig**: Reference to a ConfigMap or Secret that contains Spinnaker configuration files coming from Halyard. Only `configMap` is currently supported. This configuration is described [below](#spinnakerconfig).
+Note that you can use spinsvc for brevity. You can also use `spinnakerservices.spinnaker.io`.
+
+### .spec.spinnakerConfig
+
+Reference to a ConfigMap or Secret that contains Spinnaker configuration files coming from Halyard. Only `configMap` is currently supported. This configuration is described [below](#spinnakerconfig).
```

Comment:
This paragraph is no longer true. `.spec.spinnakerConfig.config` contains the configuration you can find in Halyard under an entry in `deploymentConfigurations`